src/misc.cpp

```diff
@@ -782,7 +782,7 @@ void SearchBox::event_cursor_moved(QMouseEvent *e) { move(e->x(), e->y()); }
 
 void SearchBox::event_xbutton_clicked()
 {
-    QKeyEvent ke(QEvent::KeyPress, Qt::Key_Escape, 0); // temp..
+    QKeyEvent ke(QEvent::KeyPress, Qt::Key_Escape, nullptr); // temp..
```

Comment:
Isn't `Qt::NoModifier` better here, instead of `nullptr`? The c-tor is:

```c++
QKeyEvent(QEvent::Type type, int key, Qt::KeyboardModifiers modifiers, const QString &text = QString(), bool autorep = false, ushort count = 1)
```

Reply:
Of course.
And there's some more mistakes. 
Will update it and ping you.

Reply:
> And there's some more mistakes.

I didn't find any.

> Will update it and ping you.

OK.

Reply:
I was tricked by the variable name: `size`.... turns out that it's a pointer.
My first pass was right. Updating.